packages/manager/modules/web-paas/src/components/modify-plan/index.js

```diff
@@ -0,0 +1,31 @@
+import angular from 'angular';
```

Comment:
this is not required. we discussed about it. you can delete this file and folder